docs/pipelines.md

```diff
@@ -942,17 +942,17 @@ Sharing `Results` between `Tasks` in a `Pipeline` happens via
 a `Result` and another receives it as a `Parameter` with a variable such as
 `$(tasks.<task-name>.results.<result-name>)`. Pipeline support two new types of
 results and parameters: array `[]string` and object `map[string]string`.
-Both are alpha features and can be enabled by setting `enable-api-fields` to `alpha`.
+Object result is alpha feature and can be enabled by setting `enable-api-fields` to `alpha`.
```

Comment:
NIT: shouldn't we say that "array results are a beta feature. They can be enabled in the `v1` API by setting the `enable-api-fields` to `beta`

Reply:
I am not sure, with `v1beta1` API, for this feature to work, `enable-api-fields` does not have to be `alpha` anymore which means it can be set to either `beta` or `stable`. 

@chitrangpatel how did you approach this when we promoted propagated params/workspaces  features? 🤔 Or these features were not classified as `beta`? will appreciate any guidance. Thanks!

Reply:
Propagated Params was moved straight to stable since stable and beta were same at the time.
Propagated workspaces is currently beta meaning that it will work if `enable-api-fields` is `alpha` or `beta` but **not** `stable`.

Reply:
yea, we should clarify that array results is an beta feature

@pritidesai it seems that in both v1 and v1beta1, we check that beta feature flag is enabled -- @lbernick we may need clarification in the TEP about whether the flag should be the same in both versions

Reply:
> Propagated workspaces is currently beta meaning that it will work if enable-api-fields is alpha or beta but not stable.

Thanks @chitrangpatel for getting back. How did we document that propagated workspace is `beta` feature? At the same time, I am checking docs. I just want to make sure its consistent across many different features.

Reply:
https://github.com/tektoncd/pipeline/pull/6030/files is all I had done.

Reply:
So I just checked the PR #6030, there is no specific guidelines other than the replacing `alpha` with `beta` in the docs 🙃 I will update the docs here but it's really very confusing for folks using `v1beta1` APIs.